Dockerfile

```diff
@@ -45,7 +45,8 @@ RUN apt-get update && apt-get install -y \
 	protobuf-compiler \
 	protobuf-c-compiler \
 	python-protobuf \
-	&& mkdir -p /usr/src/criu \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
1. since there are no directories in `/var/lib/apt/lists/`, option `-r` look redundant.
2. is there any benefit in removing those files, if they are not appearing in the resulting image?

Reply:
Sorry i was wrong about "no directories"